src/Validation/Validation.php

```diff
@@ -330,12 +330,12 @@ public static function comparison($check1, string $operator, $check2): bool
                 }
                 break;
             case static::COMPARE_EQUAL:
-                if ($check1 === $check2) {
+                if ($check1 == $check2) {
```

Comment:
How did that happen?

Reply:
I think it likely happened a long time ago when we changed all the `==` to `===` and didn't notice this.